File: tutorial/05-arrays/problems/13-frequencies-in-limited-array/soft.java
import java.util.List;
import java.util.ArrayList;
class Solution {
    public List<Integer> frequencyCount(int[] arr) {
        // code here
        int length=arr.length;
        int []counter=new int[length];
        List<Integer> result=new ArrayList<>();
        for(int i=0;i<length;i++){
            counter[arr[i]-1]+=1;
        }
        for(int i=0;i<length;i++)
        {
            result.add(counter[i]);
        }
        return result;
    }
}

File: tutorial/07-sorting/problems/07-union/soft.cpp
#include<iostream>
#include<vector>
using namespace std;
class Solution {
    public:
      vector<int> findUnion(vector<int> &a, vector<int> &b) {
          // code here
          vector<int> result;;
          int l1=a.size();
          int l2=b.size();
          int i=0,j=0;
          while(i<l1 && j<l2){
              if(i>0 && a[i]==a[i-1]){
                  i++;
                  continue;
              }
              if(j<0 && b[j]==b[j-1]){
                  j++;
                  continue;
              }
              if(a[i]<b[j]){
                  result.push_back(a[i]);
                      i++;
              }
              else if(a[i]>b[j]){
                  result.push_back(b[j]);
                      j++;
              }
              else{
                  result.push_back(a[i]);
                  i++;
                  j++;
              }
          }
          while(i<l1){
               if(i>0 && a[i]==a[i-1]){
                  i++;
                  continue;
              }
              result.push_back(a[i]);
          }
           while(j<l2){
               if(j>0 && b[j]==b[j-1]){
                  j++;
                  continue;
              }
              result.push_back(b[j]);
          }
          return result;
      }
  };
  int main(){
    Solution soft;
    
    return 0;
  }